src/Symfony/Component/Notifier/Bridge/AllMySms/composer.json

```diff
@@ -0,0 +1,30 @@
+{
+    "name": "symfony/allmysms-notifier",
+    "type": "symfony-bridge",
+    "description": "Symfony AllMySms Notifier Bridge",
+    "keywords": ["sms", "allMySms", "notifier"],
+    "homepage": "https://symfony.com",
+    "license": "MIT",
+    "authors": [
+        {
+            "name": "Quentin Dequippe",
+            "email": "quentin@dequippe.tech"
+        },
+        {
+            "name": "Symfony Community",
+            "homepage": "https://symfony.com/contributors"
+        }
+    ],
+    "require": {
+        "php": ">=7.2.5",
+        "symfony/http-client": "^4.3|^5.0",
+        "symfony/notifier": "^5.3"
```

Comment:
```suggestion
        "symfony/notifier": "~5.3.0"
```

Reply:
`^5.3` is correct.

Reply:
Ok 👍  I am a bit confused, but yes you are right 👍

Reply:
Yes, this was kind-of a back and forth here. Sorry for the confusion.